py/README.md

```diff
@@ -226,6 +226,7 @@ has:
 * a `foreign_keys` field containing a list of zero or more names of fields for foreign keys to related tables
 * fields for the names of columns on the table itself, which can be referenced in column filters (e.g. `Person.person_id`)
 * zero or more fields for referencing columns on related tables (e.g. `Person.gender_concept`)
+* for cohort tables, `standard_concept_id_column` and `source_concept_id_column` fields storing the names of columns for the standard and source concepts for the domain
```

Comment:
I've already forgotten what standard vs source concept means. Do we have this documented elsewhere?

Reply:
It's an OMOP thing. For each domain, there's a specific vocabulary which is treated as the "standard" vocabulary... SNOMED in most cases... concepts in that vocabulary are used in the "foo_concept_id" columns in the OMOP schema. There are other vocabularies (e.g. ICD9, ICD10) which can be used to code data originally; when these are used, they are filled in the "foo_source_concept_id" column in the schema, and the standard concept that they map to is filled in for the "foo_concept_id" column. This way you can search for say, "hypothyroidism", using some standard concept, and find matches that originally were coded in other source vocabularies; or you can look specifically for "hypothyroidism" as defined in a source vocabulary if you want.

We normally are going to encourage researchers to use standard concepts, but our API should support searching on either.

This is not really documented in our current documentation anywhere; it's kind of fundamental to how the OMOP schema works. I'll think about the right place to do that, and we can add a link here in future.

My hope is that we're going to wind up using UIs that generate code or concept sets which make it unnecessary for researchers to understand whether they are dealing with a standard concept or not. (We could have APIs that just let you provide concept IDs and search on both standard and source concepts using them, but that may make things significantly slower.)